extensions/iidm/src/Measurements.cpp

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include <powsybl/iidm/extensions/iidm/Measurements.hpp>
+
+#include <boost/range/adaptor/filtered.hpp>
+#include <boost/range/adaptor/indirected.hpp>
+
+#include <powsybl/iidm/Identifiable.hpp>
+#include <powsybl/iidm/extensions/iidm/Measurement.hpp>
+#include <powsybl/iidm/extensions/iidm/MeasurementAdder.hpp>
+#include <powsybl/stdcxx/memory.hpp>
+
+namespace powsybl {
+
+namespace iidm {
+
+namespace extensions {
+
+namespace iidm {
+
+Measurements::Measurements(Identifiable& identifiable) :
+    Extension(identifiable) {
+}
+
+void Measurements::add(std::unique_ptr<Measurement>&& measurement) {
+    m_measurements.emplace_back(std::move(measurement));
+}
```

Comment:
To be discussed: do we have to store Measurement in `unique_ptr`

Reply:
Using a unique_ptr is more simple because Measurement is not assignable by default because of the reference to the Measurements it contains (`Measurement::m_measurements`). This is needed in order to call m_measurement.erase(it) from 
```
Measurements::remove(const Measurement& measurement)
```

(https://stackoverflow.com/questions/21310646/stdvectoreraseitem-needs-assignment-operator-to-be-defined-for-item)